fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/federation/LinkedOrganizationsInviteFragment.kt

```diff
@@ -179,11 +177,11 @@ class LinkedOrganizationsInviteFragment : Fragment() {
   companion object {
     private val TAG: String = LinkedOrganizationsInviteFragment::class.java.simpleName
     private const val QR_SIDE = 800
-    private var CREATES_INVITATION = false
+    private var createdInvitation = false
 
     @JvmStatic
     fun newInstance(createsInvitation: Boolean): LinkedOrganizationsInviteFragment {
-      CREATES_INVITATION = createsInvitation
+      createdInvitation = createsInvitation
```

Comment:
I didn't mean to simply switch the var name to lower case, but to make it not a static constant but a fragment variable. If you leave it as static constant then the upper case it's correct

Reply:
Oh yes now I understand, I corrected it :)